packages/template-retail-react-app/config/default.js

```diff
@@ -36,6 +36,12 @@ module.exports = {
             // This differs from the siteId in commerceAPIConfig for testing purposes
             siteId: 'aaij-MobileFirst',
             isProduction: false
+        },
+        // getAppOrigin will take x-forwarded-host header into account if this is true
+        useXForwardedHost: false,
```

Comment:
🤔 Should this be part of this PR? 

The interesting thing about `x-forwarded-host` is that is a runtime/sdk level option, aka it should live outside of the `app` object in the configuration. 

Generally speaking all the options that live inside `app` are configurations specific to this particular template (in this case the retail react app). Options that effect how the sdk or runtime work should be on a higher plane.

To further nit pick on this, prefixing this option with "use" might be considered a faux pas since we are developing in react and usually thats reserved for react hooks. Any ideas on what we can change that to?

Reply:
Hey @bendvc . This is not from Jinsu work. This is about the x-forwaded-host PR I merged the other that has been reverted in develop, the feature branch is probably still out of sync. I am working a new solution [here](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2050). Will take this comment into consideration. Thanks

Reply:
I think I'm gonna go on the record and take back this comment. Reason being is `app` in the case doesn't refer to the retail-react-app directly per-say, but more of "an app that is running on mrt". It just so happens that the app we are talking about uses the react sdk. So maybe it's fine to keep it in app, we don't want to pull it up next to the ssr configs.